include/amici/edata.h

```diff
@@ -233,78 +233,74 @@ class ExpData : public SimulationParameters {
+     * @param noise_scale noise scale of observed data
      * (dimension: nt x nytrue, row-major)
      */
-    void set_observed_data_std_dev(
-        std::vector<realtype> const& observed_data_std_dev
-    );
+    void set_noise_scale(std::vector<realtype> const& noise_scale);
 
     /**
-     * @brief Set identical standard deviation for all measurements.
+     * @brief Set identical noise scale for all measurements.
      *
-     * @param stdDev standard deviation (dimension: scalar)
+     * @param sigma noise scale (dimension: scalar)
      */
-    void set_observed_data_std_dev(realtype stdDev);
+    void set_noise_scale(realtype sigma);
 
     /**
-     * @brief Set standard deviations of observed data for a
+     * @brief Set noise scales of observed data for a
      * specific observable index.
      *
-     * @param observedDataStdDev standard deviation of observed data (dimension:
+     * @param sigma noise scale of observed data (dimension:
      * nt)
      * @param iy observed data index
      */
-    void set_observed_data_std_dev(
-        std::vector<realtype> const& observedDataStdDev, int iy
-    );
+    void set_noise_scale(std::vector<realtype> const& sigma, int iy);
 
     /**
-     * @brief Set all standard deviation for a given observable index to the
+     * @brief Set all noise scale for a given observable index to the
      * input value.
      *
-     * @param stdDev standard deviation (dimension: scalar)
+     * @param sigma noise scale (dimension: scalar)
      * @param iy observed data index
      */
-    void set_observed_data_std_dev(realtype stdDev, int iy);
+    void set_noise_scale(realtype sigma, int iy);
 
     /**
-     * @brief Whether standard deviation for a measurement at
+     * @brief Whether noise scale for a measurement at
      * specified timepoint- and observable index has been set.
      *
      * @param it time index
      * @param iy observable index
-     * @return boolean specifying if standard deviation of data was set
+     * @return boolean specifying if noise scale of data was set
      */
-    bool is_set_observed_data_std_dev(int it, int iy) const;
+    bool is_set_noise_scale(int it, int iy) const;
 
     /**
-     * @brief Get measurement standard deviations.
+     * @brief Get measurement noise scales.
      *
-     * @return standard deviation of observed data
+     * @return noise scale of observed data
      */
-    std::vector<realtype> const& get_observed_data_std_dev() const;
+    std::vector<realtype> const& get_noise_scale() const;
 
     /**
-     * @brief Get pointer to measurement standard deviations.
+     * @brief Get pointer to measurement noise scales.
      *
      * @param it timepoint index
-     * @return pointer to standard deviation of observed data at index
+     * @return pointer to noise scale of observed data at index
      */
-    realtype const* get_observed_data_std_dev_ptr(int it) const;
+    realtype const* get_noise_scale_ptr(int it) const;
```

Comment:
```suggestion
    realtype const* get_noise_scales_ptr(int it) const;
```
For consistency with `get_measurements_ptr`